apps/air-discount-scheme/api/src/index.ts

```diff
@@ -1 +1,2 @@
 export { Discount } from './app/modules/discount/discount.model'
+// deploy-feature
```

Comment:
Remove comment before merge

Reply:
we really need to include changes in `apps/*/infra/*` in NX affected resolution 😮‍💨